niworkflows/interfaces/registration.py

```diff
@@ -347,8 +348,17 @@ def _post_run_hook(self, runtime):
 
 
 class EstimateReferenceImageInputSpec(BaseInterfaceInputSpec):
-    in_file = File(exists=True, mandatory=True, desc="4D EPI file")
-    sbref_file = File(exists=True, desc="Single band reference image")
+    in_file = InputMultiObject(File(exists=True),
+                               mandatory=True,
+                               desc=('4D EPI file. If multiple files '
+                                     'are provided, they are assumed '
+                                     'to represent multiple echoes '
+                                     'from the same run.'))
+    sbref_file = InputMultiObject(File(exists=True),
+                                  desc=("Single band reference image. "
+                                        "If multiple files are provided, "
+                                        "they are assumed to represent "
+                                        "multiple echoes."))
```

Comment:
Along the lines of the previous comment - perhaps add a new `Bool` input to this interface (something like `multiecho`) that allows `self.inputs.in_file` to be a list and then selects the first echo? Otherwise, identify the weird conditions (e.g., multiecho=False but list at input or multiecho=True and string at input).

Reply:
Per [this comment](https://github.com/poldracklab/niworkflows/pull/408#discussion_r331304129), should `self.inputs.in_file` and `self.inputs.sbref_file` be lists no matter what? Should I check the length of the list against the new `multiecho` parameter?